quickwit/quickwit-config/src/lib.rs

```diff
@@ -51,9 +52,28 @@ pub use source_config::{
 };
 use tracing::warn;
 
+use crate::merge_policy_config::MergePolicyConfig;
 pub use crate::quickwit_config::{
     IndexerConfig, QuickwitConfig, SearcherConfig, DEFAULT_QW_CONFIG_PATH,
 };
+use crate::source_config::serialize::{SourceConfigV0_4, VersionedSourceConfig};
+
+#[derive(utoipa::OpenApi)]
+#[openapi(components(schemas(
+    IndexingResources,
+    IndexingSettings,
+    SearchSettings,
+    RetentionPolicy,
+    MergePolicyConfig,
+    DocMapping,
+    VersionedSourceConfig,
+    SourceConfigV0_4,
+    VersionedIndexConfig,
+    IndexConfigV0_4,
+)))]
+/// The OpenAPI schemas defined within the config crate that we don't
```

Comment:
I don't understand this comment. When I look at the generated `openapi.json`, I saw all these schemas.

Reply:
I would rather expect this sentence 
"/// The OpenAPI schemas defined within the config crate that we want to expose." ?

Reply:
We probably want to expose `SourceParams` and `KafkaSourceParams`, kinesis, vec...

Reply:
This is probably badly worded, the doc is supposed to note that it's done like this because we dont want to publically expose all of these types from the rust crate, rather than the OpenAPI schema per say.

Reply:
Fixed the comments o hopefully be more explicit, I agree with `SourceParams and KafkaSourceParams, kinesis` but perhaps we do that at a later point if we plan on filling out the docs more in-depth / detailed.